src/gasPriceOracle/adapters/polygon.ts

```diff
@@ -29,15 +29,15 @@ type GasStationArgs = BaseHTTPAdapterArgs & {
 class PolygonGasStation extends BaseHTTPAdapter {
   readonly chainId: number;
 
-  constructor({ chainId = 137, host, timeout = 3000, retries = 2 }: GasStationArgs = {}) {
+  constructor({ chainId = 137, host, timeout = 1500, retries = 1 }: GasStationArgs = {}) {
```

Comment:
Dropped the timeouts; 3 seconds is a long time - especially on a total of 3 attempts!